docs/upgrade.md

```diff
@@ -184,10 +184,105 @@ find /etc \( -name "*.rpmnew" -or -name "*.rpmsave" ! -name "logrotate.cron.rpms
 
 #### 4. Reboot the host
 
-
 ## Upgrade from XenServer
 
-See the [dedicated Wiki section](https://github.com/xcp-ng/xcp/wiki/Upgrade-from-XenServer).
+This article describes how to proceed in order to convert your Citrix XenServer infrastructure into a XCP-ng infrastructure. This is also the same thing for any Citrix Hypervisor version.
+
+:::tip
+**Yes** you can do that without losing ANY settings (SR, VMs, networks) from your existing Citrix Hypervisor/XenServer infrastructure!
+:::
+
+:::warning
+* Always upgrade and reboot the pool master **FIRST**
+* If HA (High Availability) is enabled, disable it before upgrading
+* Eject CDs from your VMs before upgrading [to avoid issues](https://xcp-ng.org/forum/topic/174/upgrade-from-xenserver-7-1-did-not-work): `xe vm-cd-eject --multiple`
+* If upgrading from *XCP-ng 7.5 or lower* or from *XenServer*, **it is very important to make sure clustering is not enabled on your pool**. It's a functionality that relies on proprietary software and that is not available in XCP-ng, and having it enabled before the upgrade will lead to XAPI being unable to start due to unexpected data in the database. If it is enabled or you already upgraded, see [this comment](https://github.com/xcp-ng/xcp/issues/94#issuecomment-437838544).
+:::
+
+### Before you start
+
+* Please re-read carefully all the previous warnings
+* Need a tool to manage your XCP-ng hosts? We strongly suggest that you use [Xen Orchestra](https://xcp-ng.org/#easy-to-install), the web UI for XCP-ng. Alternatively, you can use `xe` CLI or XCP-ng Center.
```

Comment:
The XO link here is an XCP-ng download link in fact.

Reply:
The error is directly coming from the wiki :D Fixing it right now!